src/install/postInstall.ts

```diff
@@ -36,7 +36,8 @@ export default async function postInstall (
   await npmRunScript('preinstall', pkg, scriptsOpts)
   await npmRunScript('install', pkg, scriptsOpts)
   await npmRunScript('postinstall', pkg, scriptsOpts)
-  return
+
+  return !!scripts['preinstall'] || !!scripts['install'] || !!scripts['postinstall']
```

Comment:
this is too much I think. We only want to cache compilation of native libs. Some lifecycle script might do necessary things. Like husky configures git hooks

Reply:
Well, if there were an easy way to only cache that, it would be great. Is there? Perhaps we could only cache `preinstall` and `install`, and always run `postinstall`.

Again, this is what zeit claims to do, I am sticking with that.